File: solutions/beecrowd/1007/1007.r
input <- file('stdin', 'r')

a <- as.integer(readLines(input, n=1))
b <- as.integer(readLines(input, n=1))
c <- as.integer(readLines(input, n=1))
d <- as.integer(readLines(input, n=1))

result = sprintf("DIFERENCA = %d", a * b - c * d)
write(result, "")

File: solutions/beecrowd/1007/1007.clj
(ns main)

(defn main []
  (let [a (Integer/parseInt (read-line))
        b (Integer/parseInt (read-line))
        c (Integer/parseInt (read-line))
        d (Integer/parseInt (read-line))]
    (- (* a b) (* c d))))

(println "DIFERENCA =" (main))
